src/google/cloud/ndb/client.py

```diff
@@ -142,8 +148,23 @@ def context(self, cache_policy=None):
             cache_policy (Optional[Callable[[key.Key], bool]]): The
                 cache policy to use in this context. See:
                 :meth:`~google.cloud.ndb.context.Context.set_cache_policy`.
+            global_cache (Optional[_cache.GlobalCache]):
```

Comment:
Might need to put the whole path here to appease sphinx fairly inconsistent reference finder.

Reply:
I think it's actually the fact that ``_cache`` isn't autodoc'ed yet. I plant to make a docs pass before merging this.